<launch>

  <!-- <param name="/use_sim_time" value="false"/> -->
  <node pkg="rviz" type="rviz" name="rviz" 
    args="-d $(find ardrone_pose_estimation)/launch/ardrone_pose_estimation.vcg"/>

  <node pkg="ardrone_pose_estimation" type="ardrone_pose_estimation" 
    name="ardrone_pose_estimation" output="screen">
  </node>

<node pkg="hector_trajectory_server" type="hector_trajectory_server" name="hector_trajectory_server">
  <param name="target_frame_name" value="world"/>
 <!-- <param name="source_frame_name" value="body"/> -->
 <param name="source_frame_name" value="base_kf"/> 
  <param name="trajectory_update_rate" value="5"/>
  <param name="trajectory_publish_rate" value="5"/>
</node>
</launch>
